.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+
+## CHAOSScast Episode Release Checklist
+
+### Scheduling an episode
+
+* **Step 1:** Propose a topic and guest(s) \(Anyone\)
+  * [ ] **Action:** Discuss with facilitator to get approval for the episode topic and guest.
+  * [ ] **Action:** Recruit interview guest to finalize topic and content of the podcast together
+* **Step 2:** Facilitate Episode \(Facilitator\)
+  * [ ] **Action:** Send invitation email to guest to find a few potential suitable dates and times \(60-90 minutes.\)
+  * [ ] **Action:** Notify organizer of guest's date and time possibilities, and confirm one.
+  * [ ] **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit?pli=1#heading=h.cmpmu92ylw4d) and start filling in information.)
+  * [ ] **Action:** Invite panelists from the CHAOSS community who have expertise in the episode topic.
+  * [ ] **Action:** Confirm panelists 
+  * [ ] **Action:** Send calendar invitation with confirmed date of the podcast 
+    * The event has to be created in the [CHAOSScast calendar](https://calendar.google.com/calendar/ical/513r47bb756t07lp5tat5nsuj0%40group.calendar.google.com/public/basic.ics)
+      * All organizers should have full access to the calendar \(i.e., Make changes and manage sharing\)
+    * Name calendar event:🎙  CHAOSScast {Topic}
+      * Start with the microphone emoticon \(🎙 \) to signal recording meeting
+  * [ ] **Action:** Send out a [confirmation email](https://docs.google.com/document/d/1Pw8OKn-qSSkMw573BZu-EZ4BZualesdZjPbpBEKvzrc/edit) that includes the [“how we podcast” document](https://docs.google.com/document/d/1j5Ng91rrFrx3PdPejv26rBxb1P7UX-O_Q0_I9TCIya0/edit#) to guests and panelists
+  * [ ] **Action:** Update the episode planning document with recording time and zoom link
+  * [ ] **Action:** Update the [episode overview table](https://docs.google.com/document/d/1qxmC1k8SayChurIq2r91CR3r_KAThTwaNEMjA6DLGhE/edit#)
+  * [ ] **Action:** If the episode is more than two weeks out, schedule a [reminder email](https://docs.google.com/document/d/17vKzCOYvRXq5zaIlv_sb80zFapTU19dRgkx5RqRx3xE/edit) for 1 week before the episode recording
+  * [ ] **Action:** Schedule a [reminder email](https://docs.google.com/document/d/17vKzCOYvRXq5zaIlv_sb80zFapTU19dRgkx5RqRx3xE/edit) for 24-hours before the episode recording
+* **Step 3:** Finalize Content \(Organizer\)
+  * [ ] **Action:** Update the episode planning document with the main topics and questions
+* **Step 4:** Post-Episode Thank You Notes \(Georg Link\)
+  * [ ] **Action:** When the episode is scheduled for release that week, notify guest and get their mailing address
```

Comment:
This item is triggered when an episode is scheduled for release. It's a good idea to send the guest an email with the release date and a link to the episode so that they can plan to spread the word. 

In that email to the guest, we can include the "email thank you" and a link to a webform for requesting a physical thank-you card in the mail. -- we'd set the webform up to inform me (@GeorgLink ) that someone provided an mailing address and I'll send it out.

Because of the trigger, I suggest moving this check-list item to a different part of the workflow to become a part of the release process.

Reply:
@GeorgLink Do you want to set up the webform? It might be easier for you to get notified in the right places if you set it up? Then we can add the link here?

Reply:
@GeorgLink One more question - now that we have a lot more people in the CHAOSS community doing interesting things, I'm hoping to have more panelists from the community and fewer guests outside of the community. I assume we only send the thank you to guests and not chaoss community members?

Reply:
Link to Google Form: https://forms.gle/ZCCSHUF1vSEMq65r7

It lives in the CHAOSS Google Drive and I get a notification to my personal email of new submissions. The notification does not include the personal identifiable information, only a link to the spreadsheet. Therefore, no PII lives in my personal account and I'll only copy it onto the envelope or card when sending it out.

I think after sending a card, I could delete the address to not store this PII long-term. I'm considering to keep the name though to track how many cards we sent and to whom.

I would like to send Thank You notes to CHAOTICS at least once. Since this is a reboot, I would like to order new cards and send them to everyone, even CHAOTICS that already have an old card. -- Of course, only if they submit the form.